dalite_xblock/dalite_xblock.py

```diff
@@ -143,6 +153,30 @@ def is_lti_ready(self):
         """Check if this XBlock has all settings so it can connect to the LTI."""
         return all((self.launch_url, self.question_id, self.assignment_id))
 
+    def _render_student_view(self, context, in_studio):
+
+        if in_studio:
+            message = _("No question selected. Please click \"Edit\" and enter the assignment ID and question ID.")
+        else:
+            message = _("This task is not configured yet. You can't finish it right now.")
+
+        fragment = super(DaliteXBlock, self).student_view(context)
+        fragment.add_javascript(loader.load_unicode('public/js/dalite_xblock.js'))
+        fragment.initialize_js('DaliteXBlock')
+
+        if not self.is_lti_ready:
+            fragment.content = u''
+            context.update(self._get_context_for_template())
+            context.update({
+                "message": message
+            })
```

Comment:
@bradenmacdonald  Thanks for the pointers, I didn't understand what you meant by: "The passport format should say ":dalite-xblock:"", but I have added the example. 

Instead of adding  "No valid LTI passport found" I had to rephrase whole sentence to be factually correct (user needs to enter the the xblock edition and set LTI passport explicitly, even if there is only a single valid passport configured (which probably should be fixed sometime in the future). 

```
        'No valid LTI passport set. Please click Edit and select LTI passport.  '
        'If you see: "No Dalite-ng LTI Passports configured" message, then '
        'please go to Advanced Settings and ensure that the LTI Passport '
        'looks like: "<passport-id>:dalite-xblock:<long hash string>".  '
        'For example: "dalite-ng:dalite-xblock:aHR0cDovLzE5Mi4xNjguMzMuMToxMDEwMDthbHBoYTtiZXRh".  '
        'If you are unsure whether your passport is correct, or don\'t know your passport, please consult '
        'your Dalite provider.'
```